evap/evaluation/management/commands/anonymize.py

```diff
@@ -76,33 +70,39 @@ def anonymize_data(self):
             self.stdout.write("")
             raise
 
-    def randomize_textanswers(self, lorem_ipsum):
-        for textanswer in TextAnswer.objects.all():
-            textanswer.answer = self.lorem(textanswer.answer, lorem_ipsum)
-            if textanswer.original_answer:
-                textanswer.original_answer = self.lorem(textanswer.original_answer, lorem_ipsum)
-            textanswer.save()
-
-    @staticmethod
-    def lorem(text, lorem_ipsum):
-        word_count = len(text.split(' '))
-        return ' '.join(itertools.islice(itertools.cycle(lorem_ipsum), word_count))
-
-    @staticmethod
-    def generate_random_usernames(first_names, last_names):
-        user_count = UserProfile.objects.count()
-        random_usernames = set()
-        while len(random_usernames) != user_count:
-            random_usernames.add((random.choice(first_names), random.choice(last_names)))
-        return random_usernames
-
-    @staticmethod
-    def replace_usernames_emails(random_usernames):
-        for user, new_username in zip(UserProfile.objects.all(), random_usernames):
+    # Anonymizes the email templates by replacing the actual subject with a sample one
+    # and replacing the body with lorem ipsum.
+    def anonymize_email_templates(self):
+        self.stdout.write("TODO: Anonymize email templates...")
```

Comment:
same here, i would prefer to not have any stubs in the code, they have the tendency to be unused for years or be removed because the plans changed :) the next person looking into this will figure out what to do by themselves.